syntax = "proto3";

package redeHoteleira;

message BookHotelRequest {
  string hotel_name = 1;
  int32 qtd_days = 2;
  int32 qtd_guests = 3;
}

message BookHotelResponse {
  bool sucess = 1;
  int32 id = 2;
  string message = 3;
}

message CancelBookHotelRequest {
  int32 id = 1;
}

message CancelBookHotelResponse {
  bool sucess = 1;
  string message = 2;
}

service RedeHoteleira {
  rpc BookHotel(BookHotelRequest) returns (BookHotelResponse);
  rpc CancelBookHotel(CancelBookHotelRequest) returns (CancelBookHotelResponse);
}
